src/api.js

```diff
@@ -0,0 +1,13 @@
+export const API_ROOT = process.env.REACT_APP_API_ROOT
+export const API_KEY = process.env.REACT_APP_API_KEY
+
+export const params = {
+  'api-key': `${API_KEY}`
+}
```

Comment:
is there a benefit to exporting these, vs just using them locally within this file?

Reply:
Yes because they are then imported into `App.js` where the `get` function is actually called. Because of the way React's `useEffect` hook works, it is necessary to actually include the function call there.

Plus exporting these variables along with a parameters object makes it nicely modular and could be used for different types of queries.